<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="1" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="36581" started-at="2019-03-26T01:05:34Z" finished-at="2019-03-26T01:06:11Z">
    <groups>
    </groups>
    <test name="Default test" duration-ms="36581" started-at="2019-03-26T01:05:34Z" finished-at="2019-03-26T01:06:11Z">
      <class name="testcases.TC003_SampleCheckin">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:testcases.TC003_SampleCheckin@55d56113]" name="beforeSuite" is-config="true" duration-ms="1650" started-at="2019-03-26T06:35:33Z" finished-at="2019-03-26T06:35:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="setData()[pri:0, instance:testcases.TC003_SampleCheckin@55d56113]" name="setData" is-config="true" duration-ms="0" started-at="2019-03-26T06:35:34Z" finished-at="2019-03-26T06:35:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setData -->
        <test-method status="PASS" signature="beforeClass()[pri:0, instance:testcases.TC003_SampleCheckin@55d56113]" name="beforeClass" is-config="true" duration-ms="1" started-at="2019-03-26T06:35:34Z" finished-at="2019-03-26T06:35:34Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeClass -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:testcases.TC003_SampleCheckin@55d56113]" name="beforeMethod" is-config="true" duration-ms="6561" started-at="2019-03-26T06:35:34Z" finished-at="2019-03-26T06:35:41Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="ValidUsernameAndPassword()[pri:0, instance:testcases.TC003_SampleCheckin@55d56113]" name="ValidUsernameAndPassword" duration-ms="29994" started-at="2019-03-26T06:35:41Z" finished-at="2019-03-26T06:36:11Z">
          <exception class="java.lang.RuntimeException">
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException
	at utils.Reporter.reportStep(Reporter.java:60)
	at utils.Reporter.reportStep(Reporter.java:70)
	at wdMethods.SeMethods.verifyDisplayed(SeMethods.java:316)
	at pages.SampleCheckinEdit.clickSCESaveAndCloseOptions(SampleCheckinEdit.java:270)
	at testcases.TC003_SampleCheckin.ValidUsernameAndPassword(TC003_SampleCheckin.java:31)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:108)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:661)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:869)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1193)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:126)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
	at org.testng.TestRunner.privateRun(TestRunner.java:744)
	at org.testng.TestRunner.run(TestRunner.java:602)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:380)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:375)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:340)
	at org.testng.SuiteRunner.run(SuiteRunner.java:289)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1301)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1226)
	at org.testng.TestNG.runSuites(TestNG.java:1144)
	at org.testng.TestNG.run(TestNG.java:1115)
	at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:114)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- ValidUsernameAndPassword -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:testcases.TC003_SampleCheckin@55d56113]" name="afterMethod" is-config="true" duration-ms="1" started-at="2019-03-26T06:36:11Z" finished-at="2019-03-26T06:36:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterTest()[pri:0, instance:testcases.TC003_SampleCheckin@55d56113]" name="afterTest" is-config="true" duration-ms="0" started-at="2019-03-26T06:36:11Z" finished-at="2019-03-26T06:36:11Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterTest -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:testcases.TC003_SampleCheckin@55d56113]" name="afterSuite" is-config="true" duration-ms="523" started-at="2019-03-26T06:36:11Z" finished-at="2019-03-26T06:36:12Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- testcases.TC003_SampleCheckin -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
